src/qml/qgismobileapp.qml

```diff
@@ -2236,8 +2243,37 @@ ApplicationWindow {
       }
 
       Connections {
+        id: mapSettingsConnections
         target: mapCanvas.mapSettings
 
+        property bool autoReFollowLocation: false
+
+        function cancelAutoFollowLocation() {
+          toast.close();
```

Comment:
This should be something the toast does by itself, move it in the QfButton toastAction's onClicked.

Reply:
Also, when somebody clicks on the action button, the timeout function should be nullified. That'll remove the need to have the autoReFollowLocation variable.